pkg/telemetry/workflowsk8s_test.go

```diff
@@ -51,12 +55,66 @@ func TestWorkflowIdentifyPlatform(t *testing.T) {
 			provider.ClusterProviderKey:      provider.ClusterProviderUnknown,
 		}, r)
 
+		// this test does not provide an OpenShift operator Pod to the fake client, so the report should omit this key
+		_, hasOpenShiftKey := r[provider.OpenShiftVersionKey]
+		require.False(t, hasOpenShiftKey)
+
+		b, err := json.Marshal(r)
+		require.NoError(t, err)
+		fmt.Printf("%s\n", b)
```

Comment:
It's high time we fix those into a proper test but that can be done in a separate PR.